openapi/sms/v1/partner-service.yaml

```diff
@@ -376,13 +354,28 @@ components:
       oneOf:
         - $ref: '#/components/schemas/OutboundMessageBatchGlobalError'
         - $ref: '#/components/schemas/OutboundMessageBatchMessageSpecificError'
+  responses:
+    ClientConfigurationInvalidFormatResponse:
+      description: Request body is invalid and does not follow API specification.
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/InvalidFieldErrors'
+    TestConnectionInvalidFormatResponse:
```

Comment:
we only need 1 response schema here, they are both identical, how about calling it `InvalidClientConfigurationFormatResponse`?

Reply:
We were thinking about doing it like this for the sake of extensibility and code generation. But if you insist on making it uniform, I am okay with that.

Reply:
i imagine the testing of the connection setup and saving of the connection setup would always have a similar payload, or?  it's kind of like a preview of if the save would be successful

Reply:
Fair point, we will merge them.